internal/provider/provider.go

```diff
@@ -23,6 +23,7 @@ import (
 	oauthAuthServerSettingsScopesExclusiveScopes "github.com/pingidentity/terraform-provider-pingfederate/internal/resource/oauthauthserversettingsscopesexclusivescopes"
 	oauthIssuer "github.com/pingidentity/terraform-provider-pingfederate/internal/resource/oauthissuers"
 	protocolMetadataLifetimeSettings "github.com/pingidentity/terraform-provider-pingfederate/internal/resource/protocolmetadatalifetimesettings"
+	redirectValidation "github.com/pingidentity/terraform-provider-pingfederate/internal/resource/redirectvalidation"
```

Comment:
We probably don't need to rename these imports

Reply:
Imports were fixed as a part of another branch recently merged into main. These have been merged and the new structure is applied in the latest commit for this resource.